### ISSUE TITLE
Quick numeric summary returned after processing files

### ISSUE THREAD

**Issue by federicomarini** [d3b-center/annoFuse#16]
Idea: to have a quick numeric summary returned after processing files

Rationale: often one might overlook what happened and "just takes" the output. Exposing a compact summary of what was retained/filtered out can help to be aware of parameter values.

Implementation: could be a combination of `message` calls, that e.g. can be printed out to console without messing around with the returned output 😉

**Comment by kgaonkar6** [comment]
Good idea! Do you have any suggestions as to how to go about doing this efficiently?

**Comment by federicomarini** [comment]
I'll dig a little into the code of the specific function and try to give some pointers.

Otherwise, to elaborate a little more on the idea:
- dimensions of what was provided as input -> dimensions of what is output
- what happens to this input file, also with simple `message` calls
- maybe some short overview statistics on how many functions got assigned to this or this other category

**Comment by kgaonkar6** [comment]
Thanks for looking into it!
